web/src/HUDLayout.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react"
+import { storiesOf } from "@storybook/react"
+import HUDLayout from "./HUDLayout"
+import styled from "styled-components"
+import {Color, Height, SizeUnit} from "./constants"
+
+let Header = styled.header`
+  border-right: 1px dashed ${Color.white};
+  height: 100px;
+  padding: ${SizeUnit(0.5)};
+`
+
+let StickyNav = styled.nav`
+  background-color: ${Color.white};
+  color: ${Color.gray};
+  padding: ${SizeUnit(0.25)};
+`
+
+let Main = styled.main`
+  border-right: 1px dashed ${Color.white};
+  border-bottom: 1px dashed ${Color.white};
+  padding: ${SizeUnit(0.5)};
+  white-space: pre;
```

Comment:
are you sure you want this to be white-space: pre? this makes sense for, e.g., LogPane, but not so much for other kinds of main panes.